src/OpenTelemetry.Instrumentation.AspNet.TelemetryHttpModule/ActivityHelper.cs

```diff
@@ -144,6 +142,7 @@ public static void StopAspNetActivity(TextMapPropagator textMapPropagator, Activ
             AspNetTelemetryEventSource.Log.CallbackException(aspNetActivity, "OnStopped", callbackEx);
         }
 
+        aspNetActivity.Stop();
```

Comment:
please add a changelog entry with this bugfix link.
It'd be great to have a test covering this, but not a blocker.

Reply:
I've added both tests and changelog updates, tell me if there's any wording I need to change.